### PR TITLE
New Icons

### PR DESCRIPTION
**Pull request by baitmooth**
_ | _ | _ | _ | _
----- | ----- | ----- | ----- | ----- |
actual | antennapod | antimine | autotools | biglybt
birthdaydroid | boxcryptor | brackeyside | changelog | circuittrainer
conv6ations | cronograma | cuberite | drastic | filmstaden
financemanager | gardine | gpstest | helm | helmsync
hendroid | iss-detektor | kid3 | librespeed | look4sat moneymanagerex | mozcforandroid | myboy | myfiles | nclientv2
noice | oandbackupx | podcastaddict | portauthority | potatoprojectnotes
railplanner | ruler | secusoshoppinglist | shuttle | simplekeyboard
sknetrafiken | skymap | smartpack | solitairecg | spacelaunchnow
steamchat | sweb | terminalemulator | trafictransportsparisiens | units
ynab | 

collection

### CODE REVIEW

**Comment by dkanada** [comment]
Some general thoughts about the current icons are below. Mostly about making them simpler and more uniform, the general designs look fine to me.

- playing card icon should be a single rectangular card for simplicity
- the big `F` icon doesn't match the other circle sizes
- shuttle icon looks too complicated with that folded middle section (can simplify without removing the fold)
- simple steam icon should already exist and should be used for the steam chat icon
- MyBoy and the ISS icon are too complicated but I have no clue how to make them simpler
- punch out the circle in the spaceship icon
- unwrap the inner objects in the files and Cronograma icons (the cronograma icon can remove the border circle for example)

**Comment by baitmooth** [comment]
Thanks for ideas and hints.

- [x] playing card icon should be a single rectangular card for simplicity
- [x] the big `F` icon doesn't match the other circle sizes
- [x] shuttle icon looks too complicated with that folded middle section (can simplify without removing the fold)
- [x] simple steam icon should already exist and should be used for the steam chat icon
- [x]  MyBoy icon are too complicated 

- tried a drastic version

- [ ]  ISS icon are too complicated 
- [x] punch out the circle in the spaceship icon
- [ ]  unwrap the inner objects in the files and Cronograma icons (the cronograma icon can remove the border circle for example)
- uncircled the cronograma icon, and will see what can be unwrapped next.

**Comment by dkanada** [comment]
Another quick mention, I would use an existing settings icon for the AutoTools icon, and figure out a way to increase the outline for the wrench tool. It's too thin right now and won't show up on low DPI screens.

**Comment by baitmooth** [comment]
> Another quick mention, I would use an existing settings icon for the AutoTools icon, and figure out a way to increase the outline for the wrench tool. It's too thin right now and won't show up on low DPI screens.

Ok, thanks for the hint. I will try this. I will also redo Rail planner and Helm Sync which are similar with thin lines.

**Comment by baitmooth** [comment]
There are now two apps named 'Shopping List' they seem to be different apps which may be installed alongside. I renamed the one from secuso. And the question is a little bit about naming... are dashes allowed, like 'shopping-list'? Would an name like 'secusoshoppinglist' be better?

**Comment by dkanada** [comment]
I don't believe dashes are allowed so `secusoshoppinglist` would be best.

**Comment by baitmooth** [comment]
> I don't believe dashes are allowed so `secusoshoppinglist` would be best.

Done.

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -311,7 +320,7 @@
     <item component="ComponentInfo{nl.asnbank.asnbankieren/com.service2media.m2active.client.android.M2ActiveClient}" drawable="bankieren" />
 
     <!-- Banque -->
-    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.activities.ADashBoard}" drawable="banque" />
+    <item component="ComponentInfo{com.caisseepargne.android.mobilebanking/com.caisseepargne.android.mobilebanking.features.main.MainActivity}" drawable="banque" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:323]: Generally I keep both so people with old apps still get the icons.

Comment (baitmooth) [reply app/src/main/res/xml/appfilter.xml]: I get the idea. Fixed it.

**Comment by dkanada** [comment]
Wow you are really going through a lot of the requests :smile: I have a few comments, but at this point all of the icons can be merged, I'm just commenting on things that could be improved.

- HelmSync looks a bit larger than the other circles
- CircuitTrainer could maybe utilize one of the existing clock or person icons to better match the rest of the pack
- BoxCryptor might look good with less parts and larger gaps between them (maybe a lock icon somewhere)
- BiglyBT could somehow be simplified (the inner left curve also doesn't look entirely parallel)

**Comment by dkanada** [comment]
If you'd rather merge this as is though, I will be sure this gets in the next release.

**Comment by baitmooth** [comment]
> Wow you are really going through a lot of the requests smile I have a few comments, but at this point all of the icons can be merged, I'm just commenting on things that could be improved.

It's a fun way to spend time ;) while finally gaining inkscape-skills.

>     * HelmSync looks a bit larger than the other circles
>
I'm not completly happy with Helm/HelmSync but i have not figured out how to really improve it. It only looks bigger, and it is actually a few picopixel smaller. 
 
>     * CircuitTrainer could maybe utilize one of the existing clock or person icons to better match  the rest of the pack
> 
>     * BoxCryptor might look good with less parts and larger gaps between them (maybe a lock icon somewhere)
> 
>     * BiglyBT could somehow be simplified (the inner left curve also doesn't look entirely parallel)

Good ideas, thanks for that. I will put it in my tasklist for rework.

>  If you'd rather merge this as is though, I will be sure this gets in the next release.

I have no clear preference about the time of merge. Would be happy to see some soon, but no hurry about that. I take this as a moment for kind of feature freeze and looking at the issues.

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -518,6 +540,12 @@
     <!-- Change Detection -->
     <item component="ComponentInfo{com.bernaferrari.changedetection/com.bernaferrari.changedetection.MainActivity}" drawable="changedetection" />
 
+    <!-- Changelog -->
+    <item component="ComponentInfo{com.bytehamster.changelog/com.bytehamster.changelog.Main}" drawable="changelog" />
+
+    <!-- Changelog -->
+    <item component="ComponentInfo{com.bytehamster.changelog/com.bytehamster.changelog.Main}" drawable="changelog" />
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:547]: probably only want one of these :)

**Review by renuac85 — APPROVED** [review]

**Pull request merged**